client/modules/food/components/inventory/FoodItems.js

```diff
@@ -146,6 +150,25 @@ class FoodItems extends React.Component {
     this.setState({ validInput: this.getValidationState.all() && this.checkChanged() })
   }
 
+  touch = {
+    foodName: () => this.setState({ touched: { ...this.state.touched, foodName: true } }, this.validate),
+    foodCategory: () => this.setState({ touched: { ...this.state.touched, foodCategory: true } }, this.validate),
+    foodQuantity: () => this.setState({ touched: { ...this.state.touched, foodQuantity: true } }, this.validate),
+  }
+
+  /**
+   *  callback for setState when user selects an existing food name from Autosuggest
+   */
+  validateFocus = () => {
+    this.validate
```

Comment:
Did you mean to call this? It seems to behave the same either way.

Maybe something like
```js
if (this.validate()) {
  this.quantity.focus()
  ...
```